### PR TITLE
Fix: Remove arrays from router v2 inputs

### PR DESCRIPTION
**Pull request by mikeyrf**
## Description

Fixes # (issue)

## Checklist

- [ ] New Contracts have been tested
- [ ] Lint has been run
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: test/router/v2/SynapseRouterV2.Inspection.t.sol [hunk 1]
```diff
@@ -284,7 +284,8 @@ contract SynapseRouterV2InspectionTest is BasicSynapseRouterV2Test {
             rawParams: getSwapParams(address(poolNethWeth), 1, 0)
         });
 
-        SwapQuery[] memory actualQueries = router.getOriginAmountOut(tokenIn, tokenSymbols, amountIn);
+        SwapQuery[] memory actualQueries = new SwapQuery[](1);
+        actualQueries[0] = router.getOriginAmountOut(tokenIn, tokenSymbols[0], amountIn);
 
         assertEq(expectedQueries.length, actualQueries.length);
         checkSwapQueries(expectedQueries, actualQueries);
```
Comment (ChiTimesChi) [inline test/router/v2/SynapseRouterV2.Inspection.t.sol:291]: Small nit: will we ever need to compare arrays of `SwapQuery` structs with the change introduced in the PR?

**Review by ChiTimesChi — APPROVED** [review]
LGTM, outlined a single nit in the tests

File: test/router/v2/SynapseRouterV2.Inspection.t.sol (hunk 1 above)
Comment (mikeyrf) [reply test/router/v2/SynapseRouterV2.Inspection.t.sol:291]: Likely not. This is simply me trying to minimize changes in the PR

Comment (mikeyrf) [reply test/router/v2/SynapseRouterV2.Inspection.t.sol]: @ChiTimesChi should be good to go now.

**Pull request merged**